server/plugin/api.go

```diff
@@ -219,15 +220,7 @@ func (p *MatterpollPlugin) handleVote(vars map[string]string, request *model.Pos
 		return commandErrorGeneric, nil, errors.Wrap(err, "failed to save poll")
 	}
 
-	v, err := poll.GetVotedAnswer(userID)
-	if err != nil {
-		return commandErrorGeneric, nil, errors.Wrap(err, "failed to get voted answers")
-	}
-	p.API.PublishWebSocketEvent("has_voted", map[string]interface{}{
-		"user_id":       v.UserID,
-		"poll_id":       v.PollID,
-		"voted_answers": v.VotedAnswers,
-	}, &model.WebsocketBroadcast{UserId: userID})
+	p.publishPollMetadata(poll, userID)
```

Comment:
Given that this takes some time, it might even be wroth to create a go routine for it. 0/5

Reply:
Good suggestion 👍